### PR TITLE
feat: add faq bot

### PR DESCRIPTION
**Pull request by vicrep**
TODO:
- [x] i18n
- [x] Add deutsch flow or disable for DE

Upon completion of flow, and if user asks a question, it'll trigger the following event:
```
       'user_faq_question',
        {
          value: "Should I cover my mouth while coughing?"
        }
```

### CODE REVIEW

File: src/components/chatbot.tsx [hunk 1]
```diff
@@ -40,10 +51,10 @@ const StyledChatbot = styled(ReactSimpleChatbot)`
 
   .rsc-content {
     padding: 15px;
-    height: calc(100% - 30px);
+    height: calc(100% - ${props => (props.showInput ? 81 : 30)}px);
```
Comment (jaredsavage) [inline src/components/chatbot.tsx]: could we have a const for the input height? 
```
const INPUT_HEIGHT = 51
height: calc(100% - ${props => (props.showInput && INPUT_HEIGHT) + 30} px);
```

**Comment by jaredsavage** [comment]
Looking good

File: src/components/chatbot.tsx (hunk 1 above)
Comment (vicrep) [reply src/components/chatbot.tsx]: yes we can, almost did it initially but didn't know if overkill :p

**Comment by alexissmirnov** [comment]
ok to ship without DE, but need i18n

File: src/components/chatbot.tsx (hunk 1 above)
Comment (vicrep) [reply src/components/chatbot.tsx]: done

**Pull request merged**